src/main/java/mate/academy/model/Book.java

```diff
@@ -31,4 +39,14 @@ public class Book {
     private String description;
     private String coverImage;
     private boolean isDeleted;
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @ManyToMany(cascade = CascadeType.MERGE)
```

Comment:
is this cascade useful?

I mean wil you ever do something like:
```
someCategory = book.getCategories().get(some_index);
someCategory.setDescription("updated description");
bookRepo.save(book);
```

if yes - then this cascade is ok

also would it make sense to add `PERSIST` cascade?

Reply:
removed

Reply:
@ystankevych I can still see the `MERGE` cascade :)